From 69c1bd7089a2dc9d26b179a429b8793972d178ce Mon Sep 17 00:00:00 2001
From: Jan Janssen <janssen@mpie.de>
Date: Wed, 21 Aug 2024 18:32:23 +0200
Subject: [PATCH] remove cache

---
 setup.cfg | 1 -
 1 file changed, 1 deletion(-)

diff --git a/setup.cfg b/setup.cfg
index ccda056..97b3e74 100755
--- a/setup.cfg
+++ b/setup.cfg
@@ -20,7 +20,6 @@ classifiers =
 packages = find:
 install_requires = 
 	jsonpath-ng~=1.6.1
-	lru-cache<1
 	openpyxl>=3,<4
 	pandas>=2,<3
 	pydantic>=2,<3
